app/helpers/organisations_helper.rb

```diff
@@ -2,7 +2,13 @@ module OrganisationsHelper
   def sort_link(column, title = nil)
     title ||= column.titleize
     css_class = column == sort_column ? "current #{sort_direction}" : nil
-    direction = column == sort_column && sort_direction == "asc" ? "desc" : "asc"
+
+    direction = if column == "name"
```

Comment:
It is a little unclear to me what this change actually does. Can this be simplified? I tried the original versions and the tests still pass so I'm not quite clear what it is supposed to do

Reply:
Hmm, my tests fail if I revert the helper only. The intent of the change to the helper code is that: when creating a link for a column which is _not_ the currently sorted column, the default direction is ascending for the name column but is descending for the other columns.